src/Components/test/E2ETest/Microsoft.AspNetCore.Components.E2ETests.csproj

```
@@ -59,7 +59,9 @@
     <ProjectReference
       Include="..\testassets\BasicTestApp\BasicTestApp.csproj"
       Targets="Build;Publish"
-      Properties="TestTrimmedApps=true;PublishDir=$(MSBuildThisFileDirectory)$(OutputPath)trimmed\BasicTestApp\" />
+      Properties="TestTrimmedApps=true;PublishDir=$(MSBuildThisFileDirectory)$(OutputPath)trimmed\BasicTestApp\">
+        <BuildProjectReferences></BuildProjectReferences>
+    </ProjectReference>
```

I was thinking more like this:

```suggestion
      Properties="BuildProjectReferences=false;TestTrimmedApps=true;PublishDir=$(MSBuildThisFileDirectory)$(OutputPath)trimmed\BasicTestApp\" />
```